streams/src/test/java/org/apache/kafka/streams/integration/RocksDBMetricsIntegrationTest.java

```diff
@@ -145,7 +144,7 @@ private interface MetricsVerifier {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
+    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
```

Comment:
Why are we running this test parameterized

Reply:
I agree we probably don't need a parameterized test here and on the `ResetPartitionTimeIntegrationTest` above you called out, but maybe we can address in a followup PR